test/superlogger_test.rb

```diff
@@ -108,19 +108,19 @@ def output
   test 'action_controller_log_subscriber.process_action' do
     request('home/index')
 
-    fields = output[5]
+    fields = output[7]
```

Comment:
> It seems that over different Rails versions, the number of log lines emitted has changed, causing the test cases to fail.

Just to confirm — this statement refers to this change? i.e. the number of log lines increased, so the the target log line is now the 8th line instead of the 6th.

Copying log output for reference. Based on the diff, it seems like the 3rd and 6th lines here are new. 🤔 
```json
{"level":"info","ts":1716888538.9053261,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","method":"GET","path":"/home/index"}
{"level":"debug","ts":1716888538.905996,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","controller":"HomeController","action":"index","params":{}}
{"level":"debug","ts":1716888538.906894,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","sql":"SELECT \"somethings\".* FROM \"somethings\" WHERE \"somethings\".\"paper\" = ? AND \"somethings\".\"stone\" = ? ORDER BY \"somethings\".\"id\" ASC LIMIT ?","params":["123","456","1"],"duration":0.06}
{"level":"debug","ts":1716888538.9074361,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","msg":"  Rendering layout layouts/application.html.erb"}
{"level":"debug","ts":1716888538.907789,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","view":"_partial.html.erb","duration":0.01}
{"level":"debug","ts":1716888538.908093,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","view":"index.html.erb","duration":0.35}
{"level":"info","ts":1716888538.9087338,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","msg":"  Rendered layout layouts/application.html.erb (Duration: 1.0ms | Allocations: 4473)"}
{"level":"info","ts":1716888538.909135,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","view_duration":1.79,"db_duration":0.06}
{"level":"info","ts":1716888538.909513,"caller":"active_support/logger:27","request_id":"91e9bb2986bd4b0bb4cc144c25473b93","method":"GET","path":"/home/index","response_time":3.73,"status":200}
```

Reply:
Yup! These test cases were written for a much older version of Rails - the version we are using (Rails 6.1) is the earliest version that isn't already EOL. The reason I didn't use Rails 7.1 is because it seems to break our testing setup in many different ways.